google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -187,6 +193,26 @@ public static BlobSourceOption decryptionKey(String key) {
   }
 
   /**
+   * Downloads this blob to the given file path.
+   *
+   * @param path destination
+   * @throws IOException upon failure
+   */
+  public void downloadTo(Path path) throws IOException {
+    FileOutputStream outputStream = new FileOutputStream(path.toFile());
```

Comment:
Probably it is worth making this code fully `java.nio` (without mixing `nio` and `pre-nio` file writing techniques and classes). Specifically `Files.newOutputStream(path)`. I'm also not sure what should be done with buffering here. None of these methods creaet buffered streams, but looks like we don't need it, as we have buffer on the outside.

Also, please use the "try with resources" block for the stream too (not only for the reader).